docs/root/modules/snipeit/config.md

```diff
@@ -0,0 +1,11 @@
+## SnipeIT Configuration
+
+.. _SnipeIT_config:
+
+Follow these steps to analyze SnipeIT users and assets in Cartography.
+
+1. Prepare an API token for SnipeIT
+    1. Follow [SnipeIT documentation](https://snipe-it.readme.io/reference/generating-api-tokens) to generate a API token.
+    1. Populate `SNIPEIT_TOKEN` environment variable with the API token. Alternately, you can pass a different environment variable name containing the API token
```

Comment:
Not familiar with SnipeIT, but is there a specific permission that the token needs? If so, should include that in documentation for the user.

Reply:
Snipe-IT API token uses the permission of the user who generated it: https://snipe-it.readme.io/reference/api-permissions

Reply:
Would be helpful to include that in the doc but I won't block on that